src/content/learn/describing-the-ui.md

```diff
@@ -314,7 +314,7 @@ Baca **[Mengoper Props ke sebuah Komponen](/learn/passing-props-to-a-component)*
 
 </LearnMore>
 
-## Merender Secara Kondisional {/*conditional-rendering*/}
+## Me-render Secara Kondisional {/*conditional-rendering*/}
```

Comment:
```suggestion
## Me-*render* Secara Kondisional {/*conditional-rendering*/}
```

Shall we make this *italic* as well?

Reply:
Before, we have been discuss to NOT to make italic for **Heading** cc @resir014

Reply:
Yes only in heading

Reply:
Waiting for @zainfathoni  confirm

Reply:
I see. I missed that guideline. If it has been decided, that's okay, then.
Though I'm wondering, what was the basis of that decision? Is it mentioned in PUEBI or anywhere else?